jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -47,7 +47,11 @@ def parse_data(app, file_obj, data_type=None, data_label=None):
     #  supports MaNGA and JWST data.
     if isinstance(file_obj, fits.hdu.hdulist.HDUList):
         _parse_hdulist(app, file_obj, file_name=data_label)
-    elif isinstance(file_obj, str) and os.path.exists(file_obj):
+    elif isinstance(file_obj, str):
+        if file_obj.lower().endswith('.gif'):  # pragma: no cover
```

Comment:
While makes for cool demo, I cannot think of a valid science case for this one, so I didn't bother to write a test. Also a remote data download for this seems overkill.